lib/dialects/abstract/query-generator.js

```diff
@@ -2228,6 +2228,19 @@ const QueryGenerator = {
     return result ? result : '1=1';
   },
 
+  // A recursive parser for nested where conditions
+  parseConditionObject(conditions, path) {
+    path = path || [];
+    return _.reduce(conditions, (r, v, k) => { // result, key, value
```

Comment:
why not name the params `result`, `key`, `value`? More common though is `previous`, `current`, `index`
(even though I realize this is copy+pasted)